### PR TITLE
Add support for submitting governance transactions via gnosis safe service

### PR DESCRIPTION
**Pull request by asaj**
Also adds descriptions, prompts

### CODE REVIEW

File: typescript/infra/src/core/multisend.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { ChainConnection, ChainName } from '@abacus-network/sdk';
+import { types } from '@abacus-network/utils';
+
+import { getSafe, getSafeService } from '../utils/safe';
+
+export abstract class MultiSend {
```
Comment (yorhodes) [inline typescript/infra/src/core/multisend.ts:6]: im not sure about the usefulness of this versus just having extensions of ChainConnection and mapping calls through chainConnection.send but nbd

File: typescript/infra/src/core/multisend.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  readonly chain: ChainName;
+
+  constructor(chain: ChainName) {
+    super();
+    this.chain = chain;
+  }
+
+  async sendTransactions(calls: types.CallData[]) {
+    console.log(`Please submit the following manually to ${this.chain}:`);
+    console.log(calls);
```
Comment (yorhodes) [inline typescript/infra/src/core/multisend.ts:36]: maybe 
```suggestion
    console.log(JSON.stringify(calls));
```

File: typescript/infra/src/core/govern.ts [hunk 3]
```diff
@@ -55,86 +147,66 @@ export class AbacusCoreGovernor<Chain extends ChainName> {
...
+      const signer = connection.signer;
+      if (!signer) throw new Error(`no signer found`);
+      const signerAddress = await signer.getAddress();
+      const proposer = await canProposeSafeTransactions(
+        signerAddress,
+        chain,
+        connection,
+        safeAddress,
+      );
+      if (!proposer)
```
Comment (yorhodes) [inline typescript/infra/src/core/govern.ts:193]: what about if (proposer) return safe instead of this throws
and the try catch only around estimate gas?

**Review by yorhodes — APPROVED** [review]

File: typescript/infra/src/core/multisend.ts (hunk 1 above)
Comment (asaj) [reply typescript/infra/src/core/multisend.ts:6]: Yeah I don't love this either.

The rationale was that for gnosis safe, you don't actually want to map calls through `chainConnection.send` b/c you want to batch them.

There's probably a better way than this though

**Pull request merged**